doc/dev/osd_internals/erasure_coding/ecbackend.rst

```diff
@@ -49,43 +49,43 @@ Distinguished acting set positions
 ----------------------------------
 
 With the replicated strategy, all replicas of a PG are
-interchangeable.  With erasure coding, different positions in the
-acting set have different pieces of the erasure coding scheme and are
+interchangeable.  With erasure-coding, different positions in the
```

Comment:
i don't think "erasure-coding" is better than "erasure coding" here. as it is a noun not an adjective in this context. the same applies to "objectstore" versus "object-store".

Reply:
Changed all the occurrences